runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -246,8 +251,27 @@ private JLabel buildConfigButton(Config config)
 					@Override
 					public void mousePressed(MouseEvent mouseEvent)
 					{
+						editConfigButton.setIcon(CONFIG_ICON);
```

Comment:
Is this needed? When a user clicks that icon, he'll be taken to the config page, when he returns the configs should rebuild and return to their default state, am I missing something?

Reply:
If I remove this line, the cogs stay in the hover state when returning to the config panel. I can probably remove the mouseReleased() one though, that is redundant

Reply:
Ah okay, I thought it would rebuild. If you take mouse released out, it won't restore to default when the user hovers and doesn't click

Reply:
mousePressed() is when you click down the mouse, mouseReleased() is when you 'release' the click. but when you click down, it opens a new panel so mouseReleased() is never even called. So its fine :)
I think you are confusing mouseReleased() and mouseExited()